### DISCUSSION TITLE
Effects on spatial scene?

### DISCUSSION THREAD

**Discussion by Roms1383** [tesselode/kira#95] [Q&A] [answered]
Hi and kudos for the latest release it's really getting better and better!

Is there a particular reason why spatial scene and/or emitters do not support effects in the same fashion that tracks do?

Pardon my ignorance if it's obvious, I'm not from audio engineering I just happen to use kira for an existing game's alternative audio capabilities and slowly learning in the process.

**Comment by Roms1383** [comment]
So, to answer my own question I actually overlooked in the documentation that a scene listener accepts `ListenerSettings` which allow to set the [track](https://docs.rs/kira/latest/kira/spatial/listener/struct.ListenerSettings.html#method.track) to route to, indirectly allowing to apply [effects](https://docs.rs/kira/latest/kira/track/index.html#effects) on sounds coming from emitters.

**Reply by tesselode** [reply, marked as answer]
Hi, sorry I forgot to reply, but you're correct that that's how you would apply effects to everything in a spatial scene. I'm planning to add per-emitter effects in a future version of Kira.
